roles/package_management/tasks/builded_install.yml

```diff
@@ -0,0 +1,34 @@
+---
+- block:
+    - debug: msg="Removing Ideascube folders and copying new data on Ideasbook."
+    - file: path=/var/ideascube state=absent
+    - file: path=/var/cache/ideascube state=absent
+    - command: rsync -a /mnt/pkg_source/{{ generic_project_name }}/{{ item.src}} {{ item.dst }}
+      with_items:
+        - src: "var/ideascube"
+          dst: "/var/"
+        - src: "var/cache/ideascube"
+          dst: "/var/cache/"
+    - command: "chown -R {{ username }}: {{ item }}"
```

Comment:
Why not assigning a group as well? And why not using the [file module](http://docs.ansible.com/ansible/latest/modules/file_module.html)?

Reply:
"{{ username }}:" means username:primary_group_of_username

Reply:
About file module, because "recurse" : recursively set the specified file attributes (applies only to directories). But we need file as well here.